### PR TITLE
Disable SAN staking access

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase/billing/subscription.ex [hunk 1]
```diff
@@ -52,6 +52,7 @@ defmodule Sanbase.Billing.Subscription do
       :cancel_at_period_end,
       :status
     ])
+    |> foreign_key_constraint(:plan_id, name: :subscriptions_plan_id_fkey)
```
Comment (tspenov) [inline lib/sanbase/billing/subscription.ex]: do we need an explicit name here ?

File: test/sanbase/billing/subscription_test.exs [hunk 2]
```diff
@@ -2,36 +2,12 @@ defmodule Sanbase.Billing.SubscriptionTest do
...
-    conn_apikey_free = setup_apikey_auth(build_conn(), apikey_free)
-
-    {:ok,
-     conn: conn,
-     user: user,
-     product: product,
-     plan_essential: plan_essential,
-     plan_pro: plan_pro,
-     conn_apikey: conn_apikey,
-     conn_apikey_free: conn_apikey_free}
```
Comment (tspenov) [inline test/sanbase/billing/subscription_test.exs]: wow, nice cleanup here

File: test/sanbase_web/graphql/billing/subscribe_api_test.exs [hunk 3]
```diff
@@ -6,55 +6,44 @@ defmodule SanbaseWeb.Graphql.Billing.SubscribeApiTest do
...
+  alias Sanbase.Auth.User
   alias Sanbase.StripeApi
   alias Sanbase.StripeApiTestReponse
   alias Sanbase.Billing.Subscription
 
   setup_with_mocks([
-    {StripeApi, [], [create_product: fn _ -> StripeApiTestReponse.create_product_resp() end]},
-    {StripeApi, [], [create_plan: fn _ -> StripeApiTestReponse.create_plan_resp() end]},
-    {StripeApi, [],
+    {StripeApi, [:passthrough],
```
Comment (tspenov) [inline test/sanbase_web/graphql/billing/subscribe_api_test.exs]: is :passthrough really needed here. All called functions from `StripeApi` should be mocked

File: test/sanbase/billing/access_test.exs [hunk 4]
```diff
@@ -110,30 +70,36 @@ defmodule Sanbase.Billing.AccessTest do
...
       to = Timex.shift(Timex.now(), days: -10)
       query = daily_active_deposits_query(from, to)
-      result = execute_query(context.conn, query, "dailyActiveDeposits")
+      result = execute_query_with_error(context.conn, query, "dailyActiveDeposits")
 
       refute called(Sanbase.Clickhouse.DailyActiveDeposits.active_deposits(:_, from, to, :_))
-      assert result != nil
+
+      assert result =~
+               "Requested metric daily_active_deposits is not provided by the current subscription plan"
```
Comment (kalkov) [inline test/sanbase/billing/access_test.exs]: The error message can be extracted into setup

File: test/sanbase_web/graphql/blockchain/etherbi_funds_flow_api_test.exs [hunk 5]
```diff
@@ -3,32 +3,21 @@ defmodule Sanbase.Etherbi.TransactionsApiTest do
...
-        main_contract_address: contract_address,
-        token_decimals: 2
-      }
-      |> Repo.insert!()
+    %{user: user} =
+      Sanbase.Factory.insert(:subscription_premium, user: Sanbase.Factory.insert(:user))
+
+    conn = setup_jwt_auth(build_conn(), user)
+
+    exchange_address = "0x" <> Sanbase.Factory.rand_hex_str()
```
Comment (kalkov) [inline test/sanbase_web/graphql/blockchain/etherbi_funds_flow_api_test.exs]: That can be extracted into a factory or helper method `random_token_address`

File: test/sanbase_web/graphql/blockchain/etherbi_average_token_age_consumed_in_days_test.exs [hunk 6]
```diff
@@ -12,8 +12,10 @@ defmodule Sanbase.Etherbi.AverageTokenAgeConsumedInDaysApiTest do
   import Sanbase.TimescaleFactory
 
   setup do
-    staked_user = Sanbase.Factory.insert(:staked_user)
-    conn = setup_jwt_auth(build_conn(), staked_user)
+    %{user: user} =
+      Sanbase.Factory.insert(:subscription_premium, user: Sanbase.Factory.insert(:user))
```
Comment (kalkov) [inline test/sanbase_web/graphql/blockchain/etherbi_average_token_age_consumed_in_days_test.exs]: Can we use the short syntax for factory? 
```elixir
Factory.insert(:subscription_premium, user: Factory.insert(:user))
```

File: test/sanbase_web/graphql/billing/subscribe_api_test.exs (hunk 3 above)
Comment (IvanIvanoff) [reply test/sanbase_web/graphql/billing/subscribe_api_test.exs]: I got some errors, so I made it like this.

**Pull request merged**